packages/sources/dlc-btc-por/README.md

```diff
@@ -0,0 +1,55 @@
+# DLC_BTC_POR
+
+![0.0.0](https://img.shields.io/github/package-json/v/smartcontractkit/external-adapters-js?filename=packages/sources/dlc-btc-por/package.json) ![v3](https://img.shields.io/badge/framework%20version-v3-blueviolet)
+
+This document was generated automatically. Please see [README Generator](../../scripts#readme-generator) for more info.
+
+## Environment Variables
+
+| Required? |         Name          |                                        Description                                        |  Type  |             Options             |                   Default                    |
+| :-------: | :-------------------: | :---------------------------------------------------------------------------------------: | :----: | :-----------------------------: | :------------------------------------------: |
+|    ✅     |        RPC_URL        |                          The RPC URL to connect to the EVM chain                          | string |                                 |                                              |
+|    ✅     |       CHAIN_ID        |                              The EVM chain id to connect to                               | number |                                 |                   `42161`                    |
```

Comment:
debating whether we should default this env var or not as it should be in line with the RPC_URL 🤔

Reply:
I don't have any strong preference here. Obviously we can't use default for `RPC_URL` but it is known that the Arbitrum mainnet should be the destination chain.

Reply:
Given Joshua's latest comments about changes they are making, it's probably worth thinking about tradeoffs between
- A single EA that can handle RPC requests across multiple EVM chains (with multiple XXX_RPC_URLs)
- Deploying 1 EA per chain

Reply:
I think in terms of performance it would be better to run a new instance of the EA for every new chain. In that case i don't think we need to change something now or in the future. Running a single instance means that the EA needs to handle potentially multiple chains at the same time which will heavily increase the load. For example a single EA that supports arbitrum, ethereum and optimism. Considering that in all of those chains there are 10000 vaults, during a background execute cycle the EA has to fetch all the vaults, fetch 10000 BTC transactions and verify them, multiplied by 3 times as there are 3 chains. My concern is that a single background execute cycle might fail or timeout.